acestep/ui/gradio/events/training/lokr_training.py

```
@@ -0,0 +1,342 @@
+
+    checkpoints: List[Tuple[int, str]] = []
+    for d in os.listdir(checkpoint_dir):
+        if not d.startswith("epoch_"):
+            continue
+        weight_file = os.path.join(checkpoint_dir, d, "lokr_weights.safetensors")
+        if not os.path.exists(weight_file):
+            continue
+        try:
+            epoch_num = int(d.split("_")[1])
+        except Exception:
+            continue
```

Ruff (0.15.0)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)